Automattic/wp-calypso
# Help Center: use URL params for canvas state

Related to #95960

## Proposed Changes

* Use the @wordpress/router package to determine the canvas state instead of getCanvasMode which will be removed in GB 19.6.0
* Provide default value for `useLocation`, which I missed in #95964

## Why are these changes being made?


* When GB 19.6.0 is installed and active, errors are thrown in the UI and the Help Center widget is not loaded

## Testing Instructions



* Activate GB 19.6.0-rc.3 on your sandbox
* cd into apps/help-center
* Run yarn dev --sync
* Sandbox your site and widgets.wp.com
* Go to the Site Editor (Appearance -> Editor)
* Make sure help icon displays
* Make sure you can open/close the help center by clicking the icon
* Click into the editor canvas
* Make sure no UI errors related to help center appear
* Make sure the help center icon is present
* Go to the Post Editor
* Make sure the help center icon is there and is functioning
* Make sure console errors are resolved
* Make sure this has not caused regression with existing GB versions

## Pre-merge Checklist
